chunk/storage_client.go

```diff
@@ -18,6 +18,10 @@ type StorageClient interface {
 	CreateTable(name string, readCapacity, writeCapacity int64) error
```

Comment:
Agree, but this is perhaps too much scope creep. Will happily do it in a follow-up PR.